bower.json

```diff
@@ -18,6 +18,6 @@
     "site*"
   ],
   "dependencies": {
-    "localforage": "^1.4.0"
+    "localforage": "~1.4.0"
```

Comment:
I would prefer not to change this. I'm confident that the API of localforage v1 will not dramatically change. In other words, I think we will face no compatibility issues with localforage v1.5 .